.github/workflows/publish-hapi-api-protos.yml

```diff
@@ -0,0 +1,22 @@
+on:
+  release:
+    types: [published]
+
+name: hapi-api-publish
+jobs:
+  npm-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+      - run: npm ci
+      - run: npm run build-proto
+      - run: npm version patch
```

Comment:
I think you'll want to have a step that parses the version from the new tag and use `npm version <newversion>` instead and you probably want to gate this so it only runs on releases of hapi-proto

Reply:
Here's an example of gating a job, this one only runs when the api is release https://github.com/chanzuckerberg/happy/blob/0b11e16b137506cb3e93f64f9802e898d1f7a8af/.github/workflows/api-release-prod.yaml#L11-L21

Reply:
I've added gating logic and version parsing as above.